keyboards/ergodox_ez/keymaps/davidhbrown/config.h

```diff
@@ -0,0 +1,8 @@
+/*
+  Set any config.h overrides for your specific keymap here.
+  See config.h options at https://docs.qmk.fm/#/config_options?id=the-configh-file
+*/
+#undef RGBLIGHT_HUE_STEP
```

Comment:
```suggestion
#pragma once 

#undef RGBLIGHT_HUE_STEP
```

Reply:
(Thanks for the poke; I got busy with the beginning of the semester. I will try to do this this weekend.)